src/layout/_page.scss

```diff
@@ -14,3 +14,7 @@
     padding: ms(10) ms(2);
   }
 }
+
+.page-section--compact {
```

Comment:
Haven't really been chaining modifiers, but if it's something we want to consider, let's discuss.